scripts/setup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# INSTRUCTIONS:
+# This script will move the pre-push hook from script folder to
+# the .git/hooks folder
+#
+# Run the script from the oppia-android root folder:
+#
+#   bash scripts/setup.sh
```

Comment:
I've run this scrpit on windows and it works

Reply:
Thanks for checking.

Reply:
Will document that windows user need git bash command line to run this script.

Reply:
done